### DISCUSSION TITLE
Assistance Request: Lifetime Subscription Not Recognized on New Device

### DISCUSSION THREAD

**Discussion by macedothales** [aclap-dev/video-downloadhelper#335] [License & Purchase requests]
### What's your issue?

 I am reaching out to you regarding an issue I encountered while attempting to use the Video DownloadHelper application on a new computer.

Several years ago, I purchased a lifetime subscription for the Chrome extension of Video DownloadHelper. However, upon trying to utilize the application today on my new device, I was prompted to subscribe again. My understanding at the time of purchase was that the subscription was indeed lifelong.

Could you please guide me through the necessary steps to have the application recognize my lifetime subscription? I would greatly appreciate your assistance in resolving this matter promptly so that I can continue using Video DownloadHelper as intended.

Thank you for your attention to this matter. I eagerly await your response.

### The 10 first digits (only 10) of your license number or email address

macedo.thales@gmail.com

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by macedothales** [comment]
I am writing to inquire about how I can request a license key for the Video DownloadHelper application. Unfortunately, I seem to have misplaced or inadvertently deleted the email containing my license key, and I am now unable to access the full features of the application on my new device.

Could you please provide guidance on how I can retrieve or request a new license key? I would greatly appreciate any assistance you can offer in resolving this matter promptly.

Thank you very much for your attention to this request. I look forward to hearing from you soon.

**Reply by macedothales** [reply]
I am writing to inform you of an issue I encountered while attempting to recover my license key through the link provided on your website (https://www.downloadhelper.net/recover-license). Upon entering my information, I received a message stating that no license was found for VDH/Firefox. However, I would like to clarify that my license is specifically for the Chrome extension of Video DownloadHelper.

Could you please advise me on the correct procedure for requesting a license key specifically for the Chrome extension? I would greatly appreciate your assistance in resolving this matter promptly so that I can continue to enjoy the full benefits of Video DownloadHelper on my Chrome browser.

**Comment by macedothales** [comment]
I am writing to inform you of an issue I encountered while attempting to recover my license key through the link provided on your website (https://www.downloadhelper.net/recover-license). Upon entering my information, I received a message stating that no license was found for VDH/Firefox. However, I would like to clarify that my license is specifically for the Chrome extension of Video DownloadHelper.

Could you please advise me on the correct procedure for requesting a license key specifically for the Chrome extension? I would greatly appreciate your assistance in resolving this matter promptly so that I can continue to enjoy the full benefits of Video DownloadHelper on my Chrome browser.

**Reply by jcfrog** [reply]
Hi, we sent you an email.
